iphone/cli/commands/_build.js

```diff
@@ -2544,7 +2544,7 @@ iOSBuilder.prototype.determineLogServerPort = function determineLogServerPort(ne
 
 	let done = false;
 	async.whilst(
-		() => !done,
+		cb => { return cb(null, !done); },
```

Comment:
I think we can completely remove this `determineLogServerPort` function. It first checks `this.target !== 'device'` and then a few lines later checks `this.target === 'device'`. So I don't actually see an instance where this is used? What do you think @cb1kenobi